dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Trn/PreferredName.cshtml

```diff
@@ -0,0 +1,41 @@
+@page "/sign-in/trn/preferred-name"
+@model TeacherIdentity.AuthServer.Pages.SignIn.Trn.PreferredName
+@{
+    ViewBag.Title = $"Is {Model.FirstName} {Model.LastName} your preferred name?";
+}
+
+@section BeforeContent {
+    <govuk-back-link href="@LinkGenerator.TrnOfficialName()" />
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <form action="@LinkGenerator.TrnPreferredName()" method="post" asp-antiforgery="true">
+            <h1 class="govuk-heading-xl">@ViewBag.Title</h1>
+
+            <p>You can tell us what you prefer to be called if it’s different from the name on your official documents.</p>
+            <p>For example, if your name is Joanne and you preferred to be called Jo.</p>
+
+            <govuk-radios asp-for="HasPreferredName">
+                <govuk-radios-fieldset>
+                    <govuk-radios-item value="False">Yes</govuk-radios-item>
```

Comment:
Nit: would be good to get the page header within the `govuk-form-group` of the radios component like in the prototype if possible (https://find-a-lost-trn-prototype.herokuapp.com/get-an-identity/preferred-name)